### DISCUSSION TITLE
dfut is not used by anyone

### DISCUSSION THREAD

**Discussion by ghost** [community/community#63564] [GitHub Education] [answered]
### Hi everyone,

Hi everyone!

I created a repository a long time ago and lost access to my mail and don't even remember it since a lot has gone by and changed.
I want to get access or the ability to use the login dfut(https://github.com/dfut). How can I achieve this?

**Comment by ldez** [comment, marked as answer]
Hello,

if you have a problem with your GitHub account contact the [GitHub support](https://support.github.com/request).

**Reply by ghost** [reply]
Thanks
